utils/visualizer/qviz/content_loader.py

```diff
@@ -2,250 +2,261 @@
 import os
 import glob
 from typing import Optional
-
+import pandas as pd
+from deltalake import DeltaTable
 import pyarrow.parquet as pa
+from collections import defaultdict
+from qviz.cube import Cube, SamplingInfo, normalize_weight
 
 
-def process_table_delta_log(table_path: str, revision_id: str) -> (list[dict], Optional[dict]):
+# Returns metadata (dict), cubes(dict), root(cube), nodes & edges (list[dict])
+def process_table(
+    table_path: str, revision_id: str
+) -> tuple[dict, dict, Cube, list[dict]]:
     """
-    Extract valid AddFiles from table's _delta_log
-    :param table_path: path to qbeast table
-    :param revision_id: target RevisionID
-    :return: a list of AddFile dictionaries and optionally the metadata dictionary
+    Process a delta table given a path to a Qbeast table and a revision ID.
+    Example json log file content:
```

Comment:
We don't need to show all this information here.
Keep only related information, maybe also something else you find relevant, and remove everything else.